ccd-definition/FixedLists/JudgmentOnline/PaymentPlanSelection-JO-nonprod.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "ID": "JoPaymentPlanSelectionList",
+    "ListElementCode": "PAY_IN_INSTALLMENT",
```

Comment:
Here also: \"PAY_IN_INSTALMENTS\ (single L and S at the end)

Reply:
I have merged your branch into mine. The changes should be there now.